packages/mgt-components/src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -615,25 +645,47 @@ export class MgtPersonCard extends MgtTemplatedComponent {
 
     const currentSectionIndex = this._currentSection ? this.sections.indexOf(this._currentSection) : -1;
 
-    const navIcons = this.sections.map((section, i, a) => {
+    const additionalSectionTemplates = this.sections.map((section, i) => {
+      let name = section.tagName.toLowerCase();
       const classes = classMap({
         active: i === currentSectionIndex,
         'section-nav__icon': true
       });
       return html`
-        <button class=${classes} @click=${() => this.updateCurrentSection(section)}>${section.renderIcon()}</button>
+        <fluent-tab id="${name}-Tab" class=${classes}
+          slot="tab" @click=${() => this.updateCurrentSection(section)}>${section.renderIcon()}
+        </fluent-tab>
+      `;
+    });
+
+    const additionalPanelTemplates = this.sections.map((section, i) => {
+      return html`
+        <fluent-tab-panel slot="tabpanel">
+              <div class="inserted">${this._currentSection ? section.asFullView() : null}</div>
+        </fluent-tab-panel>
       `;
     });
 
     const overviewClasses = classMap({
       active: currentSectionIndex === -1,
-      'section-nav__icon': true
+      'section-nav__icon': true,
+      overviewTab: true
     });
+
     return html`
-      <button class=${overviewClasses} @click=${() => this.updateCurrentSection(null)}>
-        ${getSvg(SvgIcon.Overview)}
-      </button>
-      ${navIcons}
+        <fluent-tabs  orientation="horizontal" activeindicator  @wheel=${(e: WheelEvent) =>
+          this.handleSectionScroll(e)}> 
+              this.handleSectionScroll(e)}> 
+          this.handleSectionScroll(e)}> 
+          <fluent-tab class="${overviewClasses}" slot="tab" @click=${() => this.updateCurrentSection(null)}>
+            <div>${getSvg(SvgIcon.Overview)}</div>
+          </fluent-tab>
+          ${additionalSectionTemplates}
+          <fluent-tab-panel slot="tabpanel">
+            <div class="overviewPanel">${!this._currentSection ? this.renderOverviewSection() : null}</div>
```

Comment:
sorry about the nit pick - can we keep the same casing and change this to `overview-panel`?

Reply:
nit pick'd 😄

4ca2a4a